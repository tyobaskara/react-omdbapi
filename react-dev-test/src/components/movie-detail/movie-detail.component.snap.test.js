import React from 'react';
import renderer from 'react-test-renderer';
import MovieDetail from './movie-detail.component';

test('MovieDetail snaptest', () => {
  const props = {
    detailData: {
      Poster: 'Poster',
      Title: 'Title',
      Plot: 'Plot',
      Genre: 'Genre',
      Country: 'Country',
      Director: 'Director',
      Released: 'Released',
      Actors: 'Actors',
      imdbRating: 'imdbRating',
      Writer: 'Writer',
      Production: 'Production'
    }
  }
  const component = renderer.create(
    <MovieDetail {...props} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});