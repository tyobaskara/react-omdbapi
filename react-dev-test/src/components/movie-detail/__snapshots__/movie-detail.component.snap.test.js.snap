// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetail snaptest 1`] = `
<div
  className="movie-detail"
>
  <img
    alt="Title"
    className="poster-image"
    src="Poster"
  />
  <h1>
    Title
  </h1>
  <ul
    className="separating-info"
  >
    <li>
      Released: 
      Released
    </li>
    <li>
      imdbRating: 
      imdbRating
    </li>
    <li>
      Genre: 
      Genre
    </li>
    <li>
      Production: 
      Production
    </li>
    <li>
      Country
    </li>
  </ul>
  <br />
  <p>
    Writer: 
    Writer
  </p>
  <p>
    Director: 
    Director
  </p>
  <p>
    Actors: 
    Actors
  </p>
  <br />
  <p>
    Plot
  </p>
</div>
`;
